task/fbc-validation/0.1/fbc-validation.yaml

```diff
@@ -9,7 +9,8 @@ metadata:
   name: fbc-validation
 spec:
   description: >-
-    Validates file-based catalog (FBC) image with opm command.
+    Ensures file-based catalog (FBC) components are uniquely linted to ensure they're properly constructed as part of build pipeline.
+    To validate image in build pipeline, Skopeo is used to extract information and then contents are checked using OpenShift Operator Framework's opm CLI tool.
```

Comment:
Questions: 
1. Skopeo is used to extract information - from where? Images?
2. "then contents are checked" - The content Skopeo extracted?

We may have to update the following suggestion based on the answers to the above questions.
Suggestion: To validate the image in the build pipeline, Skopeo extracts the information from the images in the build pipeline. The OpenShift Operator Framework's opm CLI tool subsequently verifies and validates the extracted contents.

Reply:
> Suggestion: To validate the image in the build pipeline, Skopeo extracts the information from the images in the build pipeline. The OpenShift Operator Framework's opm CLI tool subsequently verifies and validates the extracted contents.

Then it would be 3 lines in total, which might be too long, I don't have any problem with that but I think I can squeeze it.

FYI: Actually, this task is tricky as it uses the results from `inspect-image` task (where we used Skopeo to generate manifest content of container images) and checks it using opm tool here.

Reply:
Suggestion (where we clearly state that this task uses results from a previous task):
"The manifest data of container images obtained previously from inspect-image task is checked using OpenShift Operator Framework's opm CLI tool."

Suggestion (where we don't state it):
The same string that's currently in place.

@gtrivedi88 which one do you like?

Reply:
@dheerajodha This makes perfect sense - "The manifest data of container images obtained previously from inspect-image task is checked using OpenShift Operator Framework's opm CLI tool."